packages/api-svc/src/service.ts

```diff
@@ -215,9 +221,10 @@ export class Service {
 				authRequester,
 				messageConsumer
 			);
+
 			// MUST only add privileges once, the cmd handler is already doing it
-			// addPrivileges(authorizationClient as AuthorizationClient);
-			// await (authorizationClient as AuthorizationClient).bootstrap(true);
+			authorizationClient.addPrivilegesArray(SettlementPrivilegesDefinition);
```

Comment:
`addPrivilegesArray()` and `bootstrap()`  cannot be done in the api-svc, only the command-handler.-svc - please the comment in the line 225 above.